boards/arm/lora_e5_dev_board/board.cmake

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(pyocd "--target=stm32wle5jcix")
+board_runner_args(pyocd "--flash-opt=-O reset_type=hw")
+board_runner_args(pyocd "--flash-opt=-O connect_mode=under-reset")
+board_runner_args(jlink "--device=STM32WLE5JC" "--speed=4000")
+board_runner_args(stm32cubeprogrammer "--port=swd" "--reset=hw")
+
+include(${ZEPHYR_BASE}/boards/common/pyocd.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/stm32cubeprogrammer.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
```

Comment:
Could you add blackmagicprobe as well? Selfish comment... that's what I'm using most of the time. :-)

Reply:
For sure.